rust-runtime/aws-smithy-http-server-python/Cargo.toml

```diff
@@ -25,8 +25,8 @@ hyper = { version = "0.14.20", features = ["server", "http1", "http2", "tcp", "s
 num_cpus = "1.13.1"
 parking_lot = "0.12.1"
 pin-project-lite = "0.2"
-pyo3 = "0.16.5"
-pyo3-asyncio = { version = "0.16.0", features = ["tokio-runtime"] }
+pyo3 = "0.17.0"
+pyo3-asyncio = { version = "0.17.0", features = ["tokio-runtime"] }
```

Comment:
This was necessary to fix a bunch of noisy reborrow lints - is upgrading an issue? Everything compiles and all tests are passing.
@crisidev @unexge 
https://pyo3.rs/v0.17.0/migration.html#from-016-to-017